File: VoiceRecorder.Audio/IAudioPlayer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace VoiceRecorder.Audio
{
    public interface IAudioPlayer : IDisposable
    {
        void LoadFile(string path);
        void Play();
        void Stop();
        TimeSpan CurrentPosition { get; set; }
        TimeSpan StartPosition { get; set; }
        TimeSpan EndPosition { get; set; }
    }
}

File: ~VB Port/VoiceRecorder.Audio/IAudioPlayer.vb
Imports System.Text

Namespace VoiceRecorder.Audio
    Public Interface IAudioPlayer
        Inherits IDisposable
        Sub LoadFile(ByVal path As String)
        Sub Play()
        Sub [Stop]()
        Property CurrentPosition As TimeSpan
        Property StartPosition As TimeSpan
        Property EndPosition As TimeSpan
    End Interface
End Namespace
